Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN yum install -y epel-release && \
 ADD supervisord.conf /etc/supervisord.conf
 ADD make-nagios-fhservices-cfg make-nagios-commands-cfg fhservices.cfg.j2 commands.cfg.j2 /opt/rhmap/
 ADD plugins/default/* plugins/core/* /opt/rhmap/nagios/plugins/
+ADD plugins/default/lib/* plugins/core/lib/* /opt/rhmap/nagios/plugins/lib/
```

Comment:
We don't need this line, because the line above should copy everything recursively.